src/views/MyApps.vue

```diff
@@ -276,6 +311,35 @@ export default defineComponent({
       { hideLabel: true, key: 'actions' }
     ]
 
+    const analyticsCardTitle = (timeframe: Timeframe) => {
+      if (timeframe.key === TimeframeKeys.ONE_DAY) {
+        return `${helpTextVitals.summary24Hours} ${helpTextVitals.summary}`
+      } else if (timeframe.key === TimeframeKeys.THIRTY_DAY) {
+        return `${helpTextVitals.summary30Days} ${helpTextVitals.summary}`
+      }
+
+      return helpTextVitals.summary
+    }
+
+    const additionalFilter = computed(() => {
+      // Wait for datatable fetcher to populate the App Id array
```

Comment:
If you're waiting for a value, you might want to use `queryReady` on the metric provider.  This prevents it from issuing a query until it actually has meaningful filters.